src/TextField/stories/events.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { action } from '@storybook/addon-actions'
+import { withDocs } from 'storybook-readme'
+
+import { GridLayout } from '../../util'
+import { TextField } from '../../index'
+
+const md = `
+# Events
+
+The \`<TextField>\` component accepts multiple events:
+
+* \`onChange\`: When the field's value changes
+* \`onBlur\`: When the field loses focus
+* \`onFocus\`: When the field gains focus
+
+~~~js
+<TextField onChange={alert('change')} />
+<TextField onBlur={alert('blur')} />
+<TextField onFocus={alert('focus')} />
+~~~
+
+## Example
+
+<!-- STORY -->
+`
+
+export default withDocs(md, () =>
+  <GridLayout>
```

Comment:
What about just having only one example text field wired up to all of the event handlers? That way you don't have to click into the correct text field to see how generated events.

Reply:
Done!